symmetric-core/src/test/java/org/jumpmind/symmetric/service/impl/MockNodeService.java

```diff
@@ -397,4 +397,8 @@ public NodeStatus getNodeStatus(String nodeId) {
     public boolean setInitialLoadEnded(ISqlTransaction transaction, String nodeId) {
         return false;
     }
+
+    @Override
+    public void deleteNodeHostInstance(String nodeId, String instanceId) {
+    }
 }
```

Comment:
Just curios - is the MockNodeService class part of some unit test, which is not checked in to this branch?

Reply:
It's used in unit tests like DefaultNodeIdCreatorTest.